### PR TITLE
Audit repo

### LINKED ISSUE #277: Define and implement the audit repo
The audit service needs a repository. Let's make it

### PR DESCRIPTION
**Pull request by petereast**
Closes #277

### CODE REVIEW

**Comment by diversemix** [comment]
@erezmus / @nlisgo  ... I've approved this as it looks the right way to go. Still some work needing to be done but I'll let you have a look through it.

**Comment by erezmus** [comment]
merged master and remove [WIP]

File: audit/src/domain/audit.ts [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+  public async recordAudit(item: AuditLogItem): Promise<boolean> {
+    return this.auditRepo.putLog(item);
+  }
 }
+
+/**
+* Thoughts:
+* - How to do the thing?
+* - How to do the other thing? :thinking-face:
+*/
```
Comment (Cooryd) [inline audit/src/domain/audit.ts]: Is this comment required? 🤔

Comment (erezmus) [reply audit/src/domain/audit.ts]: probably not

File: audit/src/main.ts [hunk 2]
```diff
@@ -18,16 +27,16 @@ const setupEventBus = async (freshEventBus: EventBus) => {
...
   );
 
   // publish "ServiceStarted"
   const event: Event<ServiceStartedPayload> = {
     id: "some-wevent-id",
     created: new Date(),
     payload: {
-      type: "support",
-      name: "audit"
+      type: "support/audit",
```
Comment (Cooryd) [inline audit/src/main.ts]: Should type be stored as a variable somewhere? or is this the only place it will ever need to be declared?

Comment (erezmus) [reply audit/src/main.ts]: I think we will review this in another PR

**Pull request merged**